README.md

```diff
@@ -35,18 +40,21 @@ npm install -g ganache-cli@6.4.1
 
 Download and install python 3.6 and its respective pip and virtualenv (**python 3.7 will break things**). Then:
 
+from devtools directory:
+
 ```
 python3 -m venv venv
 source venv/bin/activate
-./devtools/install_python_requirements.sh
```

Comment:
keep a consistent pattern of 'just run the script from the dir'

Reply:
Along these lines: The invoke_tests script doesn't seem to work any more

Reply:
hmmm what error are you getting @michielderoos ?

Reply:
Figured it out, it was just a problem with my local database!